### PR TITLE
Screen play music

### PR DESCRIPTION
**Pull request by nvs266**

### CODE REVIEW

File: app/src/main/java/com/framgia/mysoundcloud/data/source/remote/FetchTrackListFromUrl.java [hunk 1]
```diff
@@ -55,11 +55,20 @@ protected void onPostExecute(List<Track> tracks) {
...
-            track.setArtworkUrl(jsonTrack.getString(Track.TrackEntity.ARTWORK_URL));
+            JSONObject jsonUser = jsonTrack.getJSONObject(Track.TrackEntity.USER);
+
+            String artworkUrl = jsonTrack.getString(Track.TrackEntity.ARTWORK_URL);
+            // Null artwork is replaced by user's avatar
+            if (artworkUrl.equals(Constant.NULL_RESULT)) {
+                artworkUrl = jsonTrack.getJSONObject(Track.TrackEntity.USER)
+                        .getString(Track.TrackEntity.AVATAR_URL);
+            }
+            track.setArtworkUrl(parseArtworkUrlToBetter(artworkUrl));
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/mysoundcloud/data/source/remote/FetchTrackListFromUrl.java]: đoạn code này bị dài quá rồi,
e tách ra thành menthod riêng nhé.

File: app/src/main/java/com/framgia/mysoundcloud/screen/musicgenres/MusicGenresFragment.java [hunk 2]
```diff
@@ -115,5 +118,8 @@ public void hideLoadingIndicator() {
 
     @Override
     public void onItemClicked(Track track) {
+        Bundle bundle = new Bundle();
+        bundle.putParcelable(Track.TrackEntity.TRACK, track);
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/mysoundcloud/screen/musicgenres/MusicGenresFragment.java]: 2.10 String constants, naming and values
Many elements of the Android such as SharedPreferences, Bundle or Intent use a key-value pair.

When using one of these components, you must define the keys as a static final :


2.11 Argument in Activity and Fragment
When passing date into an Activity or Framgment by Intent or Bundle, keys must follow the rules 2.10 and need to declare public static.

https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md

File: app/src/main/java/com/framgia/mysoundcloud/screen/playmusic/PlayMusicActivity.java [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+            default:
+                break;
+        }
+    }
+
+    private void initializeUI() {
+        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+
+        Bundle bundle = getIntent().getExtras();
+        if (bundle != null) {
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/mysoundcloud/screen/playmusic/PlayMusicActivity.java]: if bundle ==null 
return


look better

File: app/src/main/java/com/framgia/mysoundcloud/screen/playmusic/PlayMusicActivity.java [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+            mTrack = bundle.getParcelable(Track.TrackEntity.TRACK);
+
+            if (mTrack != null) {
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/mysoundcloud/screen/playmusic/PlayMusicActivity.java]: tương tự

File: app/src/main/java/com/framgia/mysoundcloud/data/source/remote/FetchTrackListFromUrl.java (hunk 1 above)
Comment (nvs266) [reply app/src/main/java/com/framgia/mysoundcloud/data/source/remote/FetchTrackListFromUrl.java]: Resolved

File: app/src/main/java/com/framgia/mysoundcloud/screen/musicgenres/MusicGenresFragment.java (hunk 2 above)
Comment (nvs266) [reply app/src/main/java/com/framgia/mysoundcloud/screen/musicgenres/MusicGenresFragment.java]: Resolved

File: app/src/main/java/com/framgia/mysoundcloud/screen/playmusic/PlayMusicActivity.java (hunk 3 above)
Comment (nvs266) [reply app/src/main/java/com/framgia/mysoundcloud/screen/playmusic/PlayMusicActivity.java]: Resolved

File: app/src/main/java/com/framgia/mysoundcloud/screen/playmusic/PlayMusicActivity.java (hunk 4 above)
Comment (nvs266) [reply app/src/main/java/com/framgia/mysoundcloud/screen/playmusic/PlayMusicActivity.java]: Resolved

**Pull request merged**